data/json/furniture_and_terrain/furniture-domestic_plants.json

```diff
@@ -137,7 +137,7 @@
     "move_cost_mod": 1,
     "required_str": 10,
     "looks_like": "t_dirtmound",
-    "flags": [ "TRANSPARENT", "PLANTABLE", "FLAT", "MOUNTABLE" ],
+    "flags": [ "TRANSPARENT", "PLANTABLE", "FLAT", "MOUNTABLE", "PLOWABLE" ],
```

Comment:
Adding this flag to the planter allow you to turn any terrain under it into a dirt mound. This flag should only be applied to terrain.

Reply:
Nice catch, thanks. I've removed the flag and changed the condition.